src/Tests/AuthoringWinUITest/AuthoringWinUITest (Package)/build/Microsoft.WinUI.AppX.targets

```diff
@@ -187,6 +205,22 @@
       </ReferenceCopyLocalPaths>
     </ItemGroup>
   </Target>
+  
+  <!-- The CreateWapProjPackageFiles task gets confused by MUI files with the same name but different sub-directory paths,
+       stripping out all but one of them. So we need to add their entries manually. -->
+  <Target Name="_IncludeMuiFilesInWapProj"

```

Comment:
I doubt much of this is needed any more with Reunion - would be best to remove it